Essentials/src/com/earth2me/essentials/commands/Commandkitreset.java

```diff
@@ -0,0 +1,66 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.CommandSource;
+import com.earth2me.essentials.Kit;
+import com.earth2me.essentials.User;
+import org.bukkit.Server;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commandkitreset extends EssentialsCommand {
+    public Commandkitreset() {
+        super ("kitreset");
+    }
+
+    @Override
+    public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+        if (user.isAuthorized("essentials.kit.reset")) {
+            final User userTo = getPlayer(server, user, args, 0);
+            final Kit kit = new Kit(args[1], ess);
+            kit.resetTime(userTo);
+            user.sendMessage(tl("kitReset", userTo.getDisplayName(), kit.getName()));
```

Comment:
But what if user == userTo? They will be sent two messages... does this happen in other parts of EssX?

Reply:
Yes it is sadly, but you can just add:
**if(userTo != user) {
userTo.sendMessage(tl("kitResetPlayer", kit.getName()));
}**